web/migrations/0056_forumvote.py

```
@@ -0,0 +1,66 @@
+                ),
+            ],
+            options={
+                "constraints": [
+                    models.CheckConstraint(
+                        condition=models.Q(
+                            models.Q(("reply__isnull", True), ("topic__isnull", False)),
+                            models.Q(("reply__isnull", False), ("topic__isnull", True)),
+                            _connector="OR",
+                        ),
+                        name="vote_topic_xor_reply",
+                    )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)